skin.aeon.nox.5/1080i/MyGames.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<window>
+	<defaultcontrol always="true">50</defaultcontrol>
+	<allowoverlay>no</allowoverlay>
+	<onload>SetProperty(Window.HasSettings,True,home)</onload>
+	<onload>ClearProperty(MediaMenu,Home)</onload>
+	<onunload>ClearProperty(Window.HasSettings,Home)</onunload>
+	<onunload>ClearProperty(Progams.Games,Home)</onunload>
+	<views>50,51,55,500</views>
+	<menucontrol>9051</menucontrol>
+	<controls>
+		<include>CommonBackground</include>
+		<control type="group">
+			<include>WindowOpenCloseAnimation</include>
+			<include condition="!Skin.HasSetting(Enable.SlimList)">AddonList</include>
+			<include condition="Skin.HasSetting(Enable.SlimList)">SlimList</include>
+			<!-- view id = 50 -->
+			<include>AddonInfoWall</include>
+			<!-- view id = 51 -->
+			<include>AddonWall</include>
+			<!-- view id = 500 -->
+		</control>
+		<control type="group">
+			<visible>Control.IsVisible(54) | Control.IsVisible(52) | Control.IsVisible(500)</visible>
+			<visible>!Window.IsActive(addoninformation)</visible>
+			<animation effect="slide" tween="cubic" time="720" start="0,806" end="0,0">VisibleChange</animation>
+			<animation effect="slide" tween="cubic" time="720" start="0,806" end="0,0">WindowOpen</animation>
+			<animation effect="slide" tween="cubic" time="720" end="0,806" start="0,0">WindowClose</animation>
+			<top>-30</top>
+			<control type="group">
+				<visible>Control.IsVisible(54) | Control.IsVisible(52)</visible>
+				<control type="group">
+					<left>410</left>
+					<top>1010</top>
+					<control type="label">
+						<width>1100</width>
+						<height>48</height>
+						<align>center</align>
+						<font>font16_title_bold</font>
+						<textcolor>white</textcolor>
+						<scroll>true</scroll>
+						<label>$INFO[ListItem.Label]</label>
+						<textoffsetx>30</textoffsetx>
+					</control>
+					<control type="label">
+						<top>45</top>
+						<width>1100</width>
+						<height>30</height>
+						<align>center</align>
+						<font>font13_title</font>
+						<textcolor>white</textcolor>
+						<label>$INFO[ListItem.Label2]</label>
+						<textoffsetx>30</textoffsetx>
+					</control>
+				</control>
+				<control type="group">
+					<top>1010</top>
+					<width>100%</width>
+					<visible>[Container.Content(albums) | Container.Content(songs)]</visible>
```

Comment:
nope, c/p ftw
That was actually copied from programs window, so it was a c/p error there in the first place. I bet it was there since the very first commit...so thx for spotting it :)